Formula/lib/libpipewire.rb

```diff
@@ -0,0 +1,52 @@
+class Libpipewire < Formula
+  desc "Server and user space API to deal with multimedia pipelines"
+  homepage "https://pipewire.org"
+  url "https://gitlab.freedesktop.org/pipewire/pipewire/-/archive/1.5.83/pipewire-1.5.83.tar.gz"
+  sha256 "f9db39d52155481c2c40e8216901ef24cdc2235725afa6cd3a79461965ffc4ac"
+  license all_of: ["GPL-2.0-only", "LGPL-2.1-or-later", "MIT"]
```

Comment:
Licenses were determined from the following files:

- https://gitlab.freedesktop.org/pipewire/pipewire/-/blob/master/LICENSE
- https://github.com/pulseaudio/pulseaudio/blob/master/LGPL
- https://jackaudio.org/api/control_8h_source.html